pyatlan/client/atlan.py

```diff
@@ -278,13 +279,15 @@ def __init__(
             start: int,
             size: int,
             assets: list[Asset],
+            aggregations: Optional[Aggregations],
```

Comment:
I'm not sure this should be at the ABC-level — only index search will allow aggregations (not lineage listing, which also derives from `SearchResults` iirc), and afaik aggregations themselves are not paged — so no real benefit to having them in the ABC, I think?